flixel/graphics/shaders/tiles/FlxColored.hx

```diff
@@ -0,0 +1,44 @@
+package flixel.graphics.shaders.tiles;
+
+import flixel.graphics.shaders.FlxBaseShader;
+
+/**
+ * Default shader used by batcher for rendering quads without textures.
+ */
+class FlxColored extends FlxBaseShader
```

Comment:
This seems like a bit of a weird class name. Wouldn't `FlxColoredShader` or `FlxColorShader` be more fitting?